lib/mpl_toolkits/tests/test_mplot3d.py

```diff
@@ -370,26 +370,37 @@ def test_plotsurface_1d_raises():
         ax.plot_surface(X, Y, z)
 
 
-def _test_proj_make_M():
+def _test_proj_make_M(proj='persp'):
     # eye point
     E = np.array([1000, -1000, 2000])
     R = np.array([100, 100, 100])
     V = np.array([0, 0, 1])
     viewM = proj3d.view_transformation(E, R, V)
-    perspM = proj3d.persp_transformation(100, -100)
```

Comment:
I am not sure about this either. This seems to assume `zfront` and `zback` refer to the coordinate on the camera's Z axis (which is backwards), but judging from other places where it's called, they seem to be equivalent to near / far plane. 
https://github.com/matplotlib/matplotlib/blob/master/lib/mpl_toolkits/mplot3d/axes3d.py#L1001